packages/clarity-js/src/layout/extract.ts

```diff
@@ -0,0 +1,67 @@
+import { Extract, Metric, Region, RegionFilter } from "@clarity-types/core";
+import { Constant } from "@clarity-types/data";
+import * as metric from "@src/data/metric";
+import * as region from "@src/layout/region";
+
+const formatRegex = /1/g;
+const digitsRegex = /[^0-9\.]/g;
+const digitsWithCommaRegex = /[^0-9\.,]/g;
+
+export function regions(root: ParentNode, value: Region[]): void {
+    for (let v of value) {
+        let valid = true;
+        let filter = v.length === 4 ? v[2] : null;
```

Comment:
Suggestion: CAn you use a desstrcutring assignment to get the components of the Region. It would be something like

const  [a, b, c, d] = v 

where a,b, c, d are the right names for hte lements of the config.

Reply:
Great point, will do.